src/einsteinpy/tests/test_symbolic/test_tensor.py

```diff
@@ -40,6 +40,23 @@ def test_Tensor_getitem():
         assert obj[p, q, r] - test_list[p][q][r] == 0
 
 
+def test_Tensor_str():
+    x, y, z = symbols("x y z")
+    test_list = [[[x, y], [y, x]], [[z, x], [y, z]]]
+    obj1 = Tensor(test_list)
+
+    assert obj1.__str__ is not object.__str__
+
+
+def test_Tensor_repr():
+    x, y, z = symbols("x y z")
+    test_list = [[[x, y], [y, sin(2 * z) - 2 * sin(z) * cos(z)]], [[z ** 2, x], [y, z]]]
+    obj1 = Tensor(test_list)
+
```

Comment:
Same here. Tell me your testing approach.

Reply:
In my first approach I wanted to use the eval() function. But the codefactor gave error because it seems it is not a secure function. So as an alternative, I used the function the repr function which used the __repr__. Maybe I could improve this test by checking if the class name is in the string

Reply:
like this 
`    machine_representation = repr(obj1)
    assert obj1.arr.__str__() in machine_representation`

Reply:
If we do `assert True` after a whole bunch of code, the tests would pass anyway. 
By default `object at 0x` appears when printing an object where `__str__` and `__repr__` is not defined. You have to make sure it does not appear.

Reply:
The tests should reflect the fact that it does not output the default thing `object at 0x.....blah blah` on doing `print(object)`

Reply:
![image](https://user-images.githubusercontent.com/51050291/60673544-4278fc00-9e78-11e9-9488-f0a05b9a4971.png)
The test reflects that there is not "object at 0x..." becuase I check if there is in the string the self.arr.__str__ is in the repr(tensor_object)